packages/toolpad-app/src/server/data.ts

```diff
@@ -570,3 +587,142 @@ export async function duplicateApp(id: string, name: string): Promise<AppMeta> {
     return newApp;
   }
 }
+
+export async function getConnections() {
+  if (config.isDemo) {
```

Comment:
Just a nit and I might have written this the same way in other functions but for stuff like this maybe we should just take one line with `if (config.isDemo) return [];`